src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -6,121 +6,124 @@
 
 namespace System.Net.Quic
 {
-    public static class QuicImplementationProviders
+    public partial class QuicClientConnectionOptions
```

Comment:
Regenerated ref sources. The `=> throw null;` were causing the PNSE generator not to generate the stub.